pulumi/infra/e2e_test_runner.py

```diff
@@ -49,8 +49,9 @@ def __init__(
                     "GRAPL_API_HOST": api.invoke_url.apply(
                         lambda url: urlparse(url).netloc
                     ),
+                    "TIMEOUT_SECS": "500",
                 },
-                timeout=60 * 15,  # 15 minutes
+                timeout=60 * 20,  # 20 minutes
```

Comment:
I believe the longest a lambda can run is 15min (900sec) https://docs.aws.amazon.com/whitepapers/latest/serverless-architectures-lambda/timeout.html

Reply:
_Sigh..._ OK.